### ISSUE TITLE
How to import a zip exportes from Agouti

### ISSUE THREAD

**Issue by vechocho** [gbif/ipt#2209]
@peterdesmet I post this here maybe you could help me I dont know what I am doing wrong

1. I just export my project 2477aa31-a8ad-4b3b-abff-15e14e43f446
2. i created a resource on the IPT v 3.0 (https://ipt.gbif-uat.org/manual/en/ipt/latest/releases)I download the version today 2023-11-20
3. on resoure type I select Material entity (as is the onle one that has sense for me) and alsu give a try to the Other type

The page shows this error
![image](https://github.com/gbif/ipt/assets/13081080/15da91a5-32b4-46fb-a7b4-974510cd62e5)

I tried to create the resource from scratch but the options that aperar for mapping are this ones.

![image](https://github.com/gbif/ipt/assets/13081080/98df8e99-87b7-4374-a288-ef0c5c917ee2)

LAs time I tried the version 3.0 there was a CamTrap option on Type and a COL option.

**Comment by peterdesmet** [comment]
Do I understand correctly that you are downloading and installing IPT 3.=

**Comment by peterdesmet** [comment]
Sorry, submitted my comment too early.

The latest version of IPT 3.0 should have a Camtrap DP option, which is what you want to use for an Agouti export. I’m don’t knoe why it would not be available. Maybe @mike-podolskiy90 can help to debug.

**Comment by mike-podolskiy90** [comment]
I would just recommend using ipt3.gbif-uat.org instead

**Comment by vechocho** [comment]
I like to use a local installation so i can play with that on my computer. 

For an account on  ipt3.gbif-uat.org should write to helpdesk?

**Comment by mike-podolskiy90** [comment]
Plenty of things has changed since the last RC release. You can try the latest snapshot if you want https://repository.gbif.org/#browse/browse:gbif:org%2Fgbif%2Fipt%2F3.0-RC2-SNAPSHOT%2F3.0-RC2-20231116.181528-51%2Fipt-3.0-RC2-20231116.181528-51.war

I can create an account for you if you give me your email address

**Comment by vechocho** [comment]
Thank you @mike-podolskiy90 I just sent you an email.

**Comment by vechocho** [comment]
IDK if i must open a new issu but i had this error while publishing the version 1 of the export

![image](https://github.com/gbif/ipt/assets/13081080/eab69ac2-3bae-417a-83ae-6c4b42b5f742)


Also once I publish the resource I want to register it so i can see how it is indexed on GBIF. but there is not option to select the institution.

**Comment by mike-podolskiy90** [comment]
Value `other` is not acceptable for `deployments#featureType` - should be one of those from the vocabulary. You can add a translation for this.

Registration for the Camtrap is currently unavailable.

**Comment by vechocho** [comment]
thanks, i made a translation. Im going to update that on the AGouti project as well but i had put other as there was not a feature for my deployment. maybe there should be an "other" option on the vocavulary?


now the error is that there is a null in filemediatype but there is no  a null on that field 

![image](https://github.com/gbif/ipt/assets/13081080/d89f3f9c-de58-4953-8abe-012e6530d3d4)

the field is image/jpeg as show below

![image](https://github.com/gbif/ipt/assets/13081080/e5c6ab36-6eb4-43da-aa66-c19263f273c1)

**Comment by mike-podolskiy90** [comment]
Please check your data. `fileMediatype` is a required field, but you don't have it for some records, for example (all `.MP4`):

```
06bceb04-5f3b-4a1e-8f27-81506c8ef2c3,ec3b02bd-bb65-4be5-ab6a-0862581a7122,activityDetection,2019-04-03T17:10:55-05:00,https://multimedia.agouti.eu/assets/06bceb04-5f3b-4a1e-8f27-81506c8ef2c3/file,true,20230702152530-101_RCNX0006.MP4,,,false,
```

I assume adding new values to the vocabulary means a new Camtrap release is required.

**Comment by peterdesmet** [comment]
@vechocho thanks for testing! The two mapping issues you discovered will be resolved in Agouti:

- featureType = `other` will be mapped to `NULL` (= empty) since the Camtrap DP vocabulary does not allow `other`
- fileMediaType files with `.MP4` will be mapped to `video/mp4`. I've updated the code so that case is ignored.

Those changes will be fixed in the next version of Agouti, so you won't see it resolved immediately.

If you discover other mapping/validation issues, please let me know.

**Comment by vechocho** [comment]
Thank you for all the help @mike-podolskiy90 

@peterdesmet on the featureType as it is not a mandatory field, maybe consider using other or let people use their own vocabulary or should I open a issue on the camtrap github? As i see now all of the features have relation with roads or places with easy access. On our deployment we put the cameras on places where we see animal crossing, or they go for salt. sometimes on the mountains on the andes we dont have trees... So i see the vocabulary is more made for forest type ecosystems and not for deserts or andean types. On our case the cameras are plced by ranger that go on places not open on the protected areas for more people.

**Comment by peterdesmet** [comment]
@vechocho yes, opening an issue in the Camtrap GitHub is preferred. I have done it for you, let's continue the conversation on extending featureType there: https://github.com/tdwg/camtrap-dp/issues/371
